source/docs/03-metadata.xml

```diff
@@ -1382,156 +1382,318 @@
+                        <p>
+                           <figure>
+                              <head>Positional attributes on folium and bifolium</head>
+                              <graphic url="../images/ExampleImages/foliadesc-02.png"/>
+                           </figure>
+                        </p>
+                        <p>On <gi scheme="MEI">bifolium</gi>, the corresponding attributes are:</p>
+                        <p>
+                           <specList>
+                              <specDesc key="att.bifoliumSurfaces" atts="outer.recto"/>
+                              <specDesc key="att.bifoliumSurfaces" atts="inner.verso"/>
+                              <specDesc key="att.bifoliumSurfaces" atts="inner.recto"/>
+                              <specDesc key="att.bifoliumSurfaces" atts="outer.verso"/>
+                           </specList>
+                        </p>
+                        <p>With those attributes, page numbers can be derived from <gi scheme="MEI">foliaDesc</gi>, alongside the information where the content on a given <gi scheme="MEI">surface</gi> is placed on a (bi)folium. Coming back to the example above this might look like so:</p>
+                        <p>
+                           <figure>
+                              <head/>
+                              <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve" valid="feasible"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/foliadesc-02.txt" parse="text"/></egXML>
+                           </figure>
+                        </p>
+                     </div>
+                     <div xml:id="foliumdimensions" type="div4">
+                        <head>Specifying page dimensions</head>
+                        <p>Within <gi scheme="MEI">surface</gi>, each <gi scheme="MEI">graphic</gi> element may specify its dimensions using the following attributes:</p>
+                        <p>
+                           <specList>
+                              <specDesc key="att.height" atts="height"/>
+                              <specDesc key="att.width" atts="width"/>
+                           </specList>
+                        </p>
+                        <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> may provide the dimensions of the original source in physical units, such as centimetres or inches. This makes it possible to combine separate parts of a manuscripts stored in different libraries, which are scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
```

Comment:
```suggestion
                        <p>The values of those attributes, however, specify the height and width of the digital resource, the scan of the source, and they are typically given in pixels (see <ptr target="#facsimileElements"/>). In contrast, <gi scheme="MEI">folium</gi> and <gi scheme="MEI">bifolium</gi> may provide the dimensions of the original source in physical units, such as centimetres or inches. This makes it possible to compare separate parts of a manuscripts stored in different libraries, which may be scanned at different resolutions. In case of <gi scheme="MEI">bifolium</gi> elements, these dimensions apply to the <hi rend="italic">folded</hi> sheet.</p>
```

Reply:
Mentioned in https://github.com/music-encoding/metadata-ig/issues/25